plugins/header_rewrite/factory.cc

```diff
@@ -139,6 +139,8 @@ condition_factory(const std::string &cond)
     c = new ConditionInbound();
   } else if (c_name == "SSN-TXN-COUNT") {
     c = new ConditionSessionTransactCount();
+  } else if (c_name == "TCPINFO") {
```

Comment:
For consistency with other conditions, I think a name like

```
TCP-INFO
```

would be preferable. But I don't feel strongly.